File: weather.java
import java.util.Scanner;

public class weather {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Is there a storm nearby: ");
        String storm = scanner.nextLine();

        if (storm.equalsIgnoreCase("yes")) {
            System.out.println("Probaly don't go outside today");
        }

        string yea;
        int cyea = Integer.parseInt(yea);
        else if (storm.equalsIgnoreCase("no")) {
            System.out.println("What is the temp in celcius");

            if(cyea <= -5) {
               System.out.println("Bbbbbrrrrrrrrr, probably don\'t go outside and if you do be sure to really bundle up wear 2-3 down jackets");

            }
            else if(cyea <= 0) {
                System.out.println("Layer up ")
            }
        }
        
    }
}

File: weather.cpp
#include <iostream>
#include <string>

int main() {
    std::string data;
    std::string yea;


    std::cout << "Is there a storm or rain near you: " << std::endl;
    std::cin >> data;
    


    if(data == "yes") {
        std::cout << "Maybe skip the walk";
        std::cout << " or take an uber" << std::endl;
    }
    else if(data == "no") {
        std::cout << "What's the temp in celcius: " << std::endl;
        std::cin >> yea;

        int cyea = std::stoi(yea);
        std::cout << "You entered:" << cyea << std::endl;

        if (cyea <= -2) {
            std::cout << "Bbbbbbbrrrrrrrr, think about wearing a down jacket/s and multiple sweaters take a subway defanitly do not walk" << std::endl;
        }
        else if (cyea <= 0) {
            std::cout << "Wear a down jacket if you have to but probably don\'t go out" << std::endl;

        }
        else if (cyea <= 10) {
            std::cout << "Very chilly, bundle up take an uber or even the subway" << std::endl;

        }
        else if(cyea < 25) {
            std::cout << "Little chilly out there, get a light jacket or/and a sweater, walk or take the subway" << std::endl;

        }
        else if(cyea >= 25) {
            std::cout << " Perfect weather! Take a walk or a subway or even the bus" << std::endl;

        }

    }
    return 0;
}

// No browser!!!